src/main/java/seedu/address/model/reminder/ReminderScheduler.java

```diff
@@ -0,0 +1,80 @@
+package seedu.address.model.reminder;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.temporal.ChronoUnit;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Logger;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.Model;
+/**
+ * Represents a thread that manages the reminders.
```

Comment:
nit: This and other comments still say `ReminderManager`.

Reply:
Okay! Will fix